src/components/Card.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable react/prop-types */
```

Comment:
Please do not disable ESlint rules.

Reply:
Hello brother thanks for your review, in this test card , i disabled it because in my local machine everything works fine just in the web it shows error

Reply:
@mr-abdellah Please Google how to handle `PropTypes` as it is an integral part of the React workflow.